sample_rules/monitored_condition.yaml

```diff
@@ -0,0 +1,39 @@
+# We start the process by unconditionally emitting a signal, transmission_gear,
+# set to 'reverse'
+- emit:
+    signal: transmission_gear
```

Comment:
It'd be much better to use dot notation for identifiers in new sample rules files.